### PR TITLE
changelog wording

### PR DESCRIPTION
**Pull request by thiolliere**
better wording for changelog

### CODE REVIEW

**Comment by thiolliere** [comment]
I kind of think linking to PR is not really useful, ppl should rather look at the rust doc,

In the future though we can link to PR.

**Review by dvdplm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [2.0.0] - 2021-01-26
 
 ### Added
-- `Decode::skip` is introduced, allowing to skip some encoded types.
-- `Decode::encoded_fixed_size` is introduced, allowing to get the fixed encoded size of a type.
+- `Decode::skip` allows to skip some encoded types.
```
Comment (dvdplm) [inline CHANGELOG.md:10]:
```suggestion
- `Decode::skip` allows to skip some encoded types. PR #201 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+- `Decode::encoded_fixed_size` allows to get the fixed encoded size of a type.
```
Comment (dvdplm) [inline CHANGELOG.md:11]:
```suggestion
- `Decode::encoded_fixed_size` allows to get the fixed encoded size of a type. PR #245 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 - `Error` now contains a chain of causes. This full error description can also be activated on
   no std using the feature `chain-error`.
```
Comment (dvdplm) [inline CHANGELOG.md:13]:
```suggestion
- `Error` now contains a chain of causes. This full error description can also be activated on
  no std using the feature `chain-error`. PR #242 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-- `Encode::encoded_size` is introduced, allowing to get the encoded size of a type more efficiently.
+- `Encode::encoded_size` allows to get the encoded size of a type more efficiently.
```
Comment (dvdplm) [inline CHANGELOG.md:14]:
```suggestion
- `Encode::encoded_size` allows to get the encoded size of a type more efficiently. PR #245 
```

File: CHANGELOG.md (hunk 1 above)
Comment (thiolliere) [reply CHANGELOG.md:10]: I guess the PR should be more https://github.com/paritytech/parity-scale-codec/pulls?q=is%3Apr+is%3Aclosed

File: CHANGELOG.md (hunk 2 above)
Comment (thiolliere) [reply CHANGELOG.md:11]: this is also 243 I think
```suggestion
- `Decode::encoded_fixed_size` allows to get the fixed encoded size of a type. PR #243
```

**Pull request merged**